PSKoans/Koans/Introduction/AboutGetMember.Koans.ps1

```diff
@@ -166,7 +182,7 @@ Describe "Get Member" {
 
             $Reason = $MethodString -f $MethodName, $CmdletName
             & (Get-Command -Name $CmdletName) |
-                Get-Member -MemberType Property -Name $MethodName |
+                Get-Member -MemberType Method -Name $MethodName |
```

Comment:
Yikes, good catch there.

Reply:
Yeah, I was confused for a bit if I did something wrong until it hit me.